src/EnergyPlus/HVACUnitaryBypassVAV.cc

```diff
@@ -4153,7 +4153,7 @@ namespace HVACUnitaryBypassVAV {
                                                                 HeatCoilLoad,
                                                                 CBVAV(CBVAVNum).HeatCoilIndex,
                                                                 QCoilActual,
-                                                                true,
+                                                                false,
```

Comment:
So supplemental heat is set to false here.  I don't quite get why, but maybe I will as I keep reviewing.

Reply:
CBVAV does not have a supplemental heating coil so the supplemental heating coil flag should be false.